app/views/digital_collections/_home_showcase.html.erb

```diff
@@ -18,9 +18,9 @@
           <%= render partial: "home_showcase_link_list", locals: { document: @portal.collection } %>
         </div>
 
-        <% showcase_images = Array(@portal.showcase_custom_images).concat Array(@portal.showcase.documents) %>
+        <% @showcase_images = Array(@portal.showcase_custom_images).concat Array(@portal.showcase.documents) %>
```

Comment:
I'm curious why this needs to be an instance variable.

Reply:
@corylown Before changing to an instance variable, further down in the page in the `content_for` block I had tried to call `showcase_images.first` to use the first one for the Open Graph image, but would get this error:
> undefined local variable or method `showcase_images'.

Would you recommend a different solution?

Reply:
Hmmmm...off the top of my head I'm not sure. Maybe there's a way to pass a local variable into a content_for block? Probably not a big deal to do it the way you have it, although I might prefer that the variable get assigned in the controller or somewhere it's easy to find.

Reply:
Or maybe just a method on @portal?